### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by Romitou**
### Description

This Pull Request adds the default language file `french.lang`.

Considering the difficulty I had translating some key words into French to keep an equivalent meaning, I would especially like to receive some reviews from the French community and know what you think about it! Also, I thought it was appropriate to use the formal pronoun *vous* to address users. Let me know what you think too! :)

If you have any questions about a translation or suggestions for improvement, please don't hesitate!

---
**Target Minecraft Versions:**  any
**Requirements:**  none
**Related Issues:**  none

### CODE REVIEW

File: src/main/resources/lang/french.lang [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+# What null (nothing) should show up as in a string/text
+none: <none>
+
+# -- Skript --
+skript:
+	copyright: ~ créé par & © Peter Güttinger alias Njol ~
+	prefix: <gray>[<gold>Skript<gray>] # not used
+	quotes error: Utilisation incorrecte des guillemets ("). Si vous voulez utiliser des guillemets dans du texte "entre guillemets", doublez-les : "".
+	brackets error: Quantité ou emplacement des accolades incorrect. Assurez-vous que chaque accolade ouvrante corresponde à une accolade fermante.
+	invalid reload: Skript ne peut être rechargé qu'avec les commandes '/reload' de Bukkit ou '/skript reload' de Skript.
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:15]: Wouldn't it be more correct to say `avec la commande 'X' de Bukkit ou 'Y' de Skript"` ?

File: src/main/resources/lang/french.lang [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+	no scripts: Aucun script n'a été trouvé, vous devriez peut-être en écrire quelques-uns ;)
+	no errors: Tous les scripts ont été chargés sans erreur.
+	scripts loaded: %s script¦¦s¦ ont été chargés avec un total de %s déclencheur¦¦s¦ et de %s commande¦¦s¦ en %s
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:18]: Un trigger correspond à une commande donc redondant

File: src/main/resources/lang/french.lang [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+		main config: la configuration principale
+		aliases: les alias
+		script: <gold>%s<reset>
+		scripts in folder: tous les scripts dans <gold>%s<reset>
+		x scripts in folder: <gold>%2$s <reset>script¦¦s¦ dans <gold>%1$s<reset>
+		empty folder: <gold>%s<reset> ne contient aucun script activé.
+	enable:
+		all:
+			enabling: Activation de tous les scripts désactivés...
+			enabled: Activation et analysation réussis de tous les scripts précédemment désactivés.
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:76]: The word `analysation `does not exist in French
Perhaps it should be replaced by something like `analyse`

File: src/main/resources/lang/french.lang [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+			error: %s ¦erreur rencontrée¦erreurs rencontrées¦ lors de l'analysation des scripts désactivés !
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:77]: Same as above

File: src/main/resources/lang/french.lang [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+			io error: Impossible de charger les scripts (certains scripts ont peut-être déjà été renommés et seront activés au redémarrage du serveur) : %s
+		single:
+			already enabled: <gold>%s<reset> est déjà activé ! Utilisez <gray>/<gold>skript <cyan>reload <red>%s<reset> pour le recharger s'il a été modifié.
+			enabling: Activation de <gold>%s<reset>...
+			enabled: <gold>%s<reset> activé et analysé avec succès.
+			error: %2$s ¦erreur rencontrée¦erreurs rencontrées¦ lors de l'analysation de <gold>%1$s<red> !
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:83]: Same as above

File: src/main/resources/lang/french.lang [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+			io error: Impossible d'activer <gold>%s<red>: %s
+		folder:
+			empty: <gold>%s<reset> ne contient aucun script désactivé.
+			enabling: Activation de <gold>%2$s <reset>script¦¦s¦ dans <gold>%1$s<reset>...
+			enabled: Activation et analysation réussis de <gold>%2$s<reset> script(s) précédemment désactivé(s) dans <gold>%1$s<reset>.
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:88]: Same as above

File: src/main/resources/lang/french.lang [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+			error: %2$s ¦erreur rencontrée¦erreurs rencontrées¦ lors de l'analysation des scripts dans <gold>%1$s<red> !
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:89]: Same as above

File: src/main/resources/lang/french.lang [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+			io error: Impossible de désactiver les scripts <gold>%s<red> (certains scripts pourraient être désactivés lors du redémarrage du serveur): %s
+	update:
+		# check/download: see Updater
+		changes:
+			# multiple versions:
+			# 	title: <gold>%s<r> update¦ has¦s have¦ been released since this version (<gold>%s<r>) of Skript:
+			# 	footer: To show the changelog of a version type <gold>/skript update changes <version><reset>
+			# invalid version: No changelog for the version <gold>%s<red> available
+			title: <bold><cyan>%s<reset> (%s)
+			next page: <grey>page %s sur %s. Utilisez <gold>/skript update changes %s<gray> pour la page suivante (astuce : utilisez la touche flèche haut)
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:111]: Wouldn't it be better to say `flèche du haut` ?

File: src/main/resources/lang/french.lang [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+	invalid argument: Argument invalide <gray>'%s<gray>'<reset>. Sont autorisés :
+	too many arguments: Cette commande ne peut accepter qu'un seul %2$s.
+	internal error: Une erreur interne s'est produite lors de la tentative d'exécution de cette commande.
+	no player starts with: Il n'y a aucun joueur en ligne dont le nom commence par '%s'
+	multiple players start with: Il y a plusieurs joueurs en ligne dont le nom commence par '%s'
+
+# -- Hooks --
+hooks:
+	hooked: Connecté avec succès à %s
+	error: Impossible de se connecter à %1$s. Cela peut se produire si Skript ne prend pas en charge la version installée de %1$s
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:155]: Same as above
A possible alternative is: `Impossible d'effectuer la liaison avec %1$s. [...]`

File: src/main/resources/lang/french.lang [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+
+# -- Aliases --
+aliases:
+	# Errors and warnings
+	empty string: '' n'est pas un item type
+	invalid item type: '%s' n'est pas un item type
+	empty name: Un alias doit avoir un nom
+	brackets error: Utilisation incorrecte d'accolades
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:163]: I think both are correct, but I find it nicer to write like this: `Utilisation incorrecte des accolades`

File: src/main/resources/lang/french.lang [hunk 11]
```diff
@@ -0,0 +1,190 @@
...
+	not enough brackets: La section commençant au caractère %s ('%s') doit être fermée
+	too many brackets: Le caractère %s ('%s') ferme une section inexistante
+	unknown variation: La variation %s n'a pas été défini avant
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:166]: I don't really know if this is better or clearer, but I give here another possibility: `La variante %s n'a pas été défini avant`

File: src/main/resources/lang/french.lang [hunk 12]
```diff
@@ -0,0 +1,190 @@
...
+	missing aliases: Les identifiants Minecraft suivants n'ont pas d'alias :
+	empty alias: L'alias n'a pas d'identifiant Minecraft ou de tags définis
+	invalid minecraft id: L'identifiant Minecraft %s est invalide
+	useless variation: La variation n'a pas d'identifiant Minecraft ou de balises spécifiés, elle est donc inutile
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:170]: Same as above: `La variante [...]`

File: src/main/resources/lang/french.lang [hunk 13]
```diff
@@ -0,0 +1,190 @@
...
+	invalid tags: Les balises spécifiées ne sont pas définies dans un JSON valide
+	unexpected section: Les sections ne sont pas autorisées ici
+	invalid variation section: Une section devrait être une section de variation, mais %s n'est pas un nom de variation valide.
```
Comment (Xamez) [inline src/main/resources/lang/french.lang:173]: Same as above: `Une section devrait être une section de variante, mais %s n'est pas un nom de variante valide.`

File: src/main/resources/lang/french.lang (hunk 3 above)
Comment (Romitou) [reply src/main/resources/lang/french.lang]: Oops, that was frenglish 😂 Thanks for catching it!

File: src/main/resources/lang/french.lang (hunk 8 above)
Comment (Romitou) [reply src/main/resources/lang/french.lang:111]: Yeah, you're right, sounds better.

File: src/main/resources/lang/french.lang [hunk 14]
```diff
@@ -0,0 +1,190 @@
...
+	correct usage: Utilisation correcte :
+	invalid argument: Argument invalide <gray>'%s<gray>'<reset>. Sont autorisés :
+	too many arguments: Cette commande ne peut accepter qu'un seul %2$s.
+	internal error: Une erreur interne s'est produite lors de la tentative d'exécution de cette commande.
+	no player starts with: Il n'y a aucun joueur en ligne dont le nom commence par '%s'
+	multiple players start with: Il y a plusieurs joueurs en ligne dont le nom commence par '%s'
+
+# -- Hooks --
+hooks:
+	hooked: Connecté avec succès à %s
```
[the remark this answers is not in the document]
Comment (Romitou) [reply src/main/resources/lang/french.lang]: Why didn't I think of this before? I'm fixing that right now, it's much better.

File: src/main/resources/lang/french.lang [hunk 15]
```diff
@@ -0,0 +1,190 @@
...
+
+# What null (nothing) should show up as in a string/text
+none: <none>
+
+# -- Skript --
+skript:
+	copyright: ~ créé par & © Peter Güttinger alias Njol ~
+	prefix: <gray>[<gold>Skript<gray>] # not used
+	quotes error: Utilisation incorrecte des guillemets ("). Si vous voulez utiliser des guillemets dans du texte "entre guillemets", doublez-les : "".
+	brackets error: Quantité ou emplacement des accolades incorrect. Assurez-vous que chaque accolade ouvrante corresponde à une accolade fermante.
```
Comment (Anarchick) [inline src/main/resources/lang/french.lang:14]: brackets error: Quantité ou emplacement des accolades incorrect. Assurez-vous que chaque accolade ouvrante '{' corresponde à une accolade fermante '}'.

File: src/main/resources/lang/french.lang [hunk 16]
```diff
@@ -0,0 +1,190 @@
...
+			io error: Impossible de désactiver les scripts <gold>%s<red> (certains scripts pourraient être désactivés lors du redémarrage du serveur): %s
+	update:
+		# check/download: see Updater
+		changes:
+			# multiple versions:
+			# 	title: <gold>%s<r> update¦ has¦s have¦ been released since this version (<gold>%s<r>) of Skript:
+			# 	footer: To show the changelog of a version type <gold>/skript update changes <version><reset>
+			# invalid version: No changelog for the version <gold>%s<red> available
+			title: <bold><cyan>%s<reset> (%s)
+			next page: <grey>page %s sur %s. Utilisez <gold>/skript update changes %s<gray> pour la page suivante (astuce : utilisez la touche flèche haut)
```
Comment (Anarchick) [inline src/main/resources/lang/french.lang:111]: next page: <grey>page %s sur %s. Utilisez <gold>/skript update changes %s<gray> pour la page suivante (Astuce : Utilisez la touche flèche haut)

File: src/main/resources/lang/french.lang [hunk 17]
```diff
@@ -0,0 +1,190 @@
...
+	info:
+		aliases: Les alias de Skript se trouvent ici : <aqua>https://github.com/SkriptLang/skript-aliases
+		documentation: La documentation de Skript se trouve ici : <aqua>https://docs.skriptlang.org/
+		tutorials: Les tutoriels de Skript se trouvent ici : <aqua>https://docs.skriptlang.org/tutorials
```
Comment (Anarchick) [inline src/main/resources/lang/french.lang:115]: tutorials: Les tutoriels de Skript se trouvent ici : <aqua>https://docs.skriptlang.org/tutorials <reset>(En anglais)

File: src/main/resources/lang/french.lang [hunk 18]
```diff
@@ -0,0 +1,190 @@
...
+# -- Updater --
+updater:
+	not started: Skript n'a pas encore vérifié la dernière version. Utilisez <gold>/skript update check<reset> pour vérifier.
+	checking: Vérification de la dernière version de Skript...
+	check in progress: La vérification d'une nouvelle version est actuellement en cours.
+	updater disabled: Le système de mise à jour étant désactivé, la vérification de la dernière version de Skript n'a pas été effectuée.
+	check error: <red>Une erreur s'est produite lors de la vérification de la dernière version de Skript :<light red> %s
+	running latest version: Vous utilisez actuellement la dernière version stable de Skript.
+	running latest version (beta): Vous utilisez actuellement une version <i>bêta<r> de Skript et aucune nouvelle version <i>stable<r> n'est disponible. Notez que vous devez mettre à jour les nouvelles versions bêta manuellement !
+	update available: Une nouvelle version de Skript est disponible : <gold>%s<reset> (vous utilisez actuellement <gold>%s<reset>)
```
Comment (Anarchick) [inline src/main/resources/lang/french.lang:130]: update available: Une nouvelle version de Skript est disponible : <gold>%s<reset> (Vous utilisez actuellement <gold>%s<reset>)

File: src/main/resources/lang/french.lang [hunk 19]
```diff
@@ -0,0 +1,190 @@
...
+	hooked: Liaison avec %s effectuée avec succès
+	error: Impossible d'effectuer la liaison avec %1$s. Cela peut se produire si Skript ne prend pas en charge la version installée de %1$s
+
+# -- Aliases --
+aliases:
+	# Errors and warnings
+	empty string: '' n'est pas un item type
+	invalid item type: '%s' n'est pas un item type
+	empty name: Un alias doit avoir un nom
+	brackets error: Utilisation incorrecte des accolades
```
Comment (Anarchick) [inline src/main/resources/lang/french.lang:163]: brackets error: Utilisation incorrecte des accolades '{' ou '}'

**Review by Anarchick — CHANGES_REQUESTED** [review]
Fixing missing Capital letter and adding symbols not present in english.lang.

File: src/main/resources/lang/french.lang (hunk 16 above)
Comment (Romitou) [reply src/main/resources/lang/french.lang:111]: Why do you want to capitalize these words?

File: src/main/resources/lang/french.lang (hunk 18 above)
Comment (Romitou) [reply src/main/resources/lang/french.lang:130]: Same remark as above, why capitalize?

File: src/main/resources/lang/french.lang (hunk 16 above)
Comment (Anarchick) [reply src/main/resources/lang/french.lang:111]: Source : https://zendictee.fr/Billet/detail/20#:~:text=Phrase%20compl%C3%A8te%20entre%20parenth%C3%A8ses,pas%20suivi%20d'un%20espace.

https://parlez-vous-francais.fr/orthographe/faut-il-mettre-une-majuscule-ou-une-minuscule-apres-deux-points/
`A` should be capitalize , but not `U`

**Comment by Weefle** [comment]
This seems to be a really good french translation, good work !

File: src/main/resources/lang/french.lang (hunk 8 above)
Comment (Xamez) [reply src/main/resources/lang/french.lang:111]: The modification has not been made

**Review by Xamez — CHANGES_REQUESTED** [review]
Other than the missing change, for me there is nothing else to change. Good job!

File: src/main/resources/lang/french.lang (hunk 8 above)
Comment (Romitou) [reply src/main/resources/lang/french.lang]: Applied, thanks for pointing that out 😊

**Review by Xamez — APPROVED** [review]

**Review by Olyno — APPROVED** [review]
Looks good to me

**Review by Anarchick — APPROVED** [review]

**Comment by Soufreur78** [comment]
En tant que français je valide également cette traduction :)

**Review by Moderocky — APPROVED** [review]
Looks fine to me. We can get a second org review on this then merge I think.

File: src/main/resources/config.sk [hunk 20]
```diff
@@ -31,7 +31,7 @@
 # ==== General Options ====
 
 language: english
-# Which language to use. Currently english, german and korean are included in the download, but custom languages can be created as well.
+# Which language to use. Currently english, german, korean and french are included in the download, but custom languages can be created as well.
```
[the remark this answers is not in the document]
Comment (Romitou) [reply src/main/resources/config.sk:34]: Alright, let's do it!

**Review by TheLimeGlass — APPROVED** [review]

**Pull request merged**